### PR TITLE
[tests] Add graphite render query to resources.coordinator

### PR DESCRIPTION
**Pull request by fishie9**
**What this PR does / why we need it**:

This PR adds the graphite render query api to resources.coordinator.

**Special notes for your reviewer**:

**Does this PR introduce a user-facing and/or backwards incompatible change?**:

```release-note

```

**Does this PR require updating code package or user-facing documentation?**:

```documentation-note

```

### CODE REVIEW

File: src/integration/resources/coordinator_client.go [hunk 1]
```diff
@@ -881,3 +882,80 @@ func toResponse(
...
+	}
+
+	defer resp.Body.Close()
+	b, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return nil, err
+	}
+
+	if status := resp.StatusCode; status != http.StatusOK {
+		return nil, fmt.Errorf("query response status not OK, received %v", status)
```
Comment (nbroyles) [inline src/integration/resources/coordinator_client.go:926]: Can we also include the body in this error message so the client has the resaon for the non-200 response.

File: src/integration/resources/coordinator_client.go [hunk 2]
```diff
@@ -881,3 +882,80 @@ func toResponse(
 
 	return nil
 }
+
+// GraphiteQuery retrieves graphite raw data.
+func (c *CoordinatorClient) GraphiteQuery(
+	graphiteReq GraphiteQueryRequest,
+) ([]Datapoint, error) {
+	if graphiteReq.From.IsZero() {
+		graphiteReq.From = time.Now().Add(-24 * time.Hour)
```
Comment (nbroyles) [inline src/integration/resources/coordinator_client.go:891]: Is this what graphite does by default (i.e. make the query window 24h if not specified)?

File: src/integration/resources/coordinator_client.go [hunk 3]
```diff
@@ -881,3 +882,80 @@ func toResponse(
...
+	}
+	if graphiteReq.Until.IsZero() {
+		graphiteReq.Until = time.Now()
+	}
+
+	queryStr := fmt.Sprintf(
+		"%s?target=%s&from=%d&until=%d",
+		graphite.ReadURL, graphiteReq.Target,
```
Comment (nbroyles) [inline src/integration/resources/coordinator_client.go:899]: Do we need to validate the Target as well? Will the query work if this is unset?

File: src/integration/resources/coordinator_client.go (hunk 2 above)
Comment (fishie9) [reply src/integration/resources/coordinator_client.go:891]: yes

File: src/integration/resources/coordinator_client.go (hunk 3 above)
Comment (fishie9) [reply src/integration/resources/coordinator_client.go:899]: The query will succeed, but the query result will be empty.
I think that's acceptable, right?

**Review by nbroyles — APPROVED** [review]

**Pull request merged**